src/Advection/tracer_advection_operators.jl

```diff
@@ -1,6 +1,24 @@
 using Oceananigans.Operators: Vᶜᶜᶜ
 using Oceananigans.Fields: ZeroField
 
+struct ThreeDimensionalTracerAdvection{N, FT, A, B, C} <: AbstractAdvectionScheme{N, FT}
```

Comment:
The name is confusing; isn't it always three-dimensional? Should we just call it `TracerAdvection`? I'm confused, this probably needs a comment...

Reply:
Changed it to `TracerAdvection` and added a docstring